Tavisca.Bootcamp.LanguageBasics.Exercise3/Program.cs

```diff
@@ -28,20 +29,517 @@ static void Main(string[] args)
             Console.ReadKey(true);
         }
 
-        private static void Test(int[] protein, int[] carbs, int[] fat, string[] dietPlans, int[] expected)
-        {
+        private static void Test(int[] protein, int[] carbs, int[] fat, string[] dietPlans, int[] expected)
+        {
             var result = SelectMeals(protein, carbs, fat, dietPlans).SequenceEqual(expected) ? "PASS" : "FAIL";
             Console.WriteLine($"Proteins = [{string.Join(", ", protein)}]");
             Console.WriteLine($"Carbs = [{string.Join(", ", carbs)}]");
-            Console.WriteLine($"Fats = [{string.Join(", ", fat)}]");
+            Console.WriteLine($"Fats = [{string.Join(", ", fat)}]");
             Console.WriteLine($"Diet plan = [{string.Join(", ", dietPlans)}]");
-            Console.WriteLine(result);
+            Console.WriteLine(result);
         }
 
-        public static int[] SelectMeals(int[] protein, int[] carbs, int[] fat, string[] dietPlans)
-        {
+        public static int[] SelectMeals(int[] protein, int[] carbs, int[] fat, string[] dietPlans)
+        {
             // Add your code here.
-            throw new NotImplementedException();
+
+
+
+
+            int[] cal = new int[protein.Length] ;
+            
+            List<int> temp_list = new List<int>();
+            List<int> new_temp_list = new List<int>();
+            int[] Answer = new int[dietPlans.Length];
+
+            for (int j = 0; j < protein.Length; j++)
+            {
+                cal[j] = (fat[j] * 9) + 5 * (protein[j] + carbs[j]);
+                
+
+            }
+            int count = 0;
+            foreach (string plan in dietPlans)
+            {
+                count += 1;
+                if (plan == "")
+                {
+                    Console.WriteLine("0");
+                    continue;
+                }
+                
+                 
+                foreach (char element in plan)
+                {
+                    switch (element)
+                    {
+                        case 'C':
+                            if (temp_list.Count == 0)

```

Comment:
If code inside switch case is too long then its better to place it in some separate method

Reply:
okay i will